WalletWasabi.Backend/wwwroot/cloudflare.html

```diff
@@ -6,43 +6,55 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in coinjoin">
     <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop, that implements trustless coinjoin over the Tor anonymity network.">
-    <meta http-equiv="onion-location" content="http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion" />
```

Comment:
why remove this?

Reply:
It doesn't work, and we are not using this file for anything.